src/System/useFeatureFlag.tsx

```diff
@@ -73,6 +73,22 @@ export const getFeatureVariant = (
   return variant
 }
 
+export const getFeatureFlag = (featureName: string): boolean | null => {
+  const featureFlags = getENV("FEATURE_FLAGS")
+  const flagEnabled = featureFlags?.[featureName]?.flagEnabled
+
+  if (flagEnabled === undefined) {
+    warnInDevelopment(
+      `[Force] Warning: cannot find ${featureName} in featureFlags: `,
+      featureFlags
+    )
+
+    return null
+  }
+
+  return flagEnabled
+}
```

Comment:
Should we update [other locations](https://github.com/search?q=repo%3Aartsy%2Fforce%20feature_flags&type=code), especially [the example doc](https://github.com/artsy/force/blob/b7fbb972adfd3dca881f3191ecd619c862e315f2/docs/unleash_a-b_testing.md?plain=1#L134), to use this?